client/app/views/models/contact.coffee

```diff
@@ -102,9 +111,9 @@ module.exports = class ContactViewModel extends Backbone.ViewModel
 
 
     onSave: ->
-        return unless @get 'new'
-        app = require 'application'
-        app.contacts.add @model
+        if @get 'new'
+            app = require 'application'
+            app.contacts.add @model
```

Comment:
maybe we should write it `require('application').contacts.add model if @get 'new'`? (I assume I'm guilty too)